ios/FluentUI/Extensions/EnvironmentValues+Extensions.swift

```diff
@@ -24,17 +24,19 @@ private extension UIEdgeInsets {
     }
 }
 
-private extension UIApplication {
-    var keyWindow: UIWindow? {
-        connectedScenes
-            .compactMap {
-                $0 as? UIWindowScene
-            }
-            .flatMap {
-                $0.windows
-            }
-            .first {
-                $0.isKeyWindow
-            }
+struct ViewControllerHolder {
+    weak var value: UIViewController?
```

Comment:
Should the SafeAreaInsets above be retrieved from the window associated with this UIViewController? (it seems to me that it should)